docs/deploying/infrastructure.md

```diff
@@ -12,28 +12,28 @@ This is managed using [builder](https://github.com/elifesciences/builder) using
 {project name}--{env}
 ```
 
-| Stack Name          | Description                                        |
-| ------------------- | -------------------------------------------------- |
-| elife-xpub--ci      | Runs tests as part of CI/CD                        |
-| elife-xpub--demo    | [Demo site](https://demo--xpub.elifesciences.org/) |
-| elife-xpub--prod    | [Production](https://xpub.elifesciences.org/)      |
-| elife-xpub--end2end | TODO                                               |
+| Stack Name          | Description                                         |
+| ------------------- | --------------------------------------------------- |
+| elife-xpub--ci      | Runs tests as part of CI/CD                         |
+| elife-xpub--staging | [Staging](https://staging--xpub.elifesciences.org/) |
+| elife-xpub--prod    | [Production](https://xpub.elifesciences.org/)       |
+| elife-xpub--end2end | TODO                                                |
```

Comment:
would move this up after `ci` and before `staging` (that's where it will be placed)